src/main/java/use_cases/PostManager.java

```diff
@@ -46,6 +43,11 @@ private boolean interactPost(Post post, Object obj, boolean type){
         return false;
     }
 
+    public Ingredient createIngridient(String pName){
+        Ingredient ing = new Ingredient(pName);
+        return ing;
+    }
+
     public void createPost(User owner, LocalDateTime postedTime, Recipe recipe, String category)
     {
```

Comment:
Sebastian, could you please also make a createRecipe method? I just realized a Recipe needs to be created before you can pass it into the new Post constructor. Check the Recipe constructor for details.